### PR TITLE
RiskTargetingCoefficientFactory

### LINKED ISSUE #100: Risk Targeting Coefficient Handler
Create handler for risk targeting coefficient endpoint. Use implementation for existing handlers as an example.

- Validates request parameters
- Submits request to factory
- Formats results (but does not round!)

### PR DESCRIPTION
**Pull request by ehunter-usgs**
fixes #100 

This includes #82, you can merge that PR first or merge this PR and close the other issue.

### CODE REVIEW

File: src/lib/risk-targeting-coefficient-handler.js [hunk 1]
```diff
@@ -0,0 +1,145 @@
...
+
+
+var _DEFAULTS;
+
+_DEFAULTS = {
+  DB_DATABASE: 'postgres',
+  DB_HOST: 'localhost',
+  DB_PASSWORD: null,
+  DB_PORT: 5432,
+  DB_SCHEMA_DETERMINISTIC: 'deterministic',
```
Comment (jmfee-usgs) [inline src/lib/risk-targeting-coefficient-handler.js]: Should this be DB_SCHEMA_RISK_TARGETING?

File: src/lib/risk-targeting-coefficient-handler.js [hunk 2]
```diff
@@ -0,0 +1,145 @@
...
+    _this = null;
+  };
+
+  _this.formatResult = function (result) {
+    return new Promise((resolve, reject) => {
+      var formatted;
+
+      try {
+        formatted = {
+          'cr1': result.mapped_cr,
```
Comment (jmfee-usgs) [inline src/lib/risk-targeting-coefficient-handler.js]: Should this be `mapped_cr1`?

**Comment by ehunter-usgs** [comment]
This includes changes to the RiskCoefficientHandler. 

Updated:
 - use `cr1` and `crs` instead of mapped_cr1 and `mapped_crs`
 - use `DB_SCHEMA` as default
 - update naming convention from `RiskTargetingCoefficient*` to `RiskCoefficient*`
 - update factory and handler to work more like existing factories and handlers

**Comment by ehunter-usgs** [comment]
Somehow this got really messed up, but I think it's correct now.

**Pull request merged**